pypeit/wavecalib.py

```diff
@@ -204,6 +205,12 @@ def build_wv_calib(self, arccen, method, skip_QA=False):
             # Sometimes works, sometimes fails
             arcfitter = autoid.HolyGrail(arccen, par=self.par, ok_mask=ok_mask)
             patt_dict, final_fit = arcfitter.get_results()
+        elif method == 'identify':
+            # Manually identify lines
+            msgs.info("Initializing the wavelength calibration tool")
+            arcfitter = identify.initialise(arccen, par=self.par)
+            final_fit = arcfitter.get_results()
+            # TODO :: If successful, maybe we should automatically generate and store the files for reidentify?
```

Comment:
Should be easy (and best) enough to modify `templates.py` to use the good solution.

Reply:
OK - this is now implemented (mostly). I haven't been able to pull the `dispname` and `dispangle` from within `wavecalib.py`, so it's set to `UNKNOWN` for now (not ideal...).

If there are any thoughts of how best to get this info from within that module, I'm all ears...